rotkehlchen/balances/manual.py

```diff
@@ -14,42 +15,52 @@
 
 
 @dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
-class ManuallyTrackedBalance:
+class _BaseManualBalance:
     identifier: int
     asset: Asset
     label: str
-    amount: FVal
     location: Location
     tags: list[str] | None
     balance_type: BalanceType
+    asset_is_missing: bool = field(default=False, metadata={'description': 'set to true when the asset points to an asset that is unknown to the db'})  # noqa: E501
```

Comment:
What is this metadata? I tried to find out more info about it but seems to say it's not part of dataclasses but just consumed by third party libraries. Why add this here? Why not just add it as a comment? Does it get used anywhere?